src/frontend/src/lib/constants/test-ids.constants.ts

```diff
@@ -24,6 +24,10 @@ export const RECEIVE_TOKENS_MODAL_BTC_MAINNET_SECTION = 'receive-tokens-modal-bt
 export const RECEIVE_TOKENS_MODAL_BTC_TESTNET_SECTION = 'receive-tokens-modal-btc-testnet-section';
 export const RECEIVE_TOKENS_MODAL_BTC_REGTEST_SECTION = 'receive-tokens-modal-btc-regtest-section';
 export const RECEIVE_TOKENS_MODAL_ETH_SECTION = 'receive-tokens-modal-eth-section';
+export const RECEIVE_TOKENS_MODAL_SOL_MAINNET_SECTION = 'receive-tokens-modal-sol-mainnet-section';
```

Comment:
Since you already added these vars, can you please add them to e2e/receive-tokens-modal.spec.ts? Otherwise, when we enable sol in the e2e env, the receive modal test will be failing due to different sol addresses in the modal on every run.

Reply:
sure, do the e2e test have the LOCAL flag to true? I added this one too

Reply:
Yes, LOCAL is true in the e2e env since we build and deploy the app the same way as we do it locally.

Reply:
I will have to comment them out until the SOLANA_NETWORK_ENABLED flag is set to true, otherwise e2e will fail